code/modules/admin/verbs/adminhelp_vr.dm

```diff
@@ -36,3 +38,37 @@
 	usr.verbs -= /client/verb/adminspice
 	spawn(6000)
 		usr.verbs += /client/verb/adminspice	// 10 minute cool-down for spice request
+*/
+
+/mob/verb/Spice(msg as text) //CHOMPStation Addition 14/1/20 Ksc - The improved request spice command.
+	set category = "Admin"
+	set name = "Request Spice"
+	set desc = "Request for admins to challange your shift"
+
+	if(say_disabled)	//This is here to try to identify lag problems
+		usr << "<font color='red'>Speech is currently admin-disabled.</font>"
```

Comment:
Standard lately is `to_chat(usr,[insert rest of code])` in place of `usr << [insert rest of code]`

Reply:
ok